src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -333,6 +352,7 @@ public static boolean positionForbidden(double armPos, double wristPos) {
         boolean ground = tip.getY() < -ARM_JOINT_TOTAL_HEIGHT;
 
         //return horizontal && vertical || ground;
+        forbFlag = true;
```

Comment:
You might want to also create a version of this in wristMovementForbidden so that the operator gets rumble on both wrist and arm movement. The forbFlag is also never unset, so the feature will currently only work once.